src/main/scala/scalismo/mesh/MeshSurfaceProperty.scala

```diff
@@ -46,7 +46,7 @@ case class ConstantProperty[@specialized(Double, Float, Int, Boolean) A](overrid
 }
 
 /** property per vertex, with interpolation */
-case class SurfacePointProperty[@specialized(Double, Float, Int, Boolean) A](override val triangulation: TriangleList, pointData: PointId => A)(implicit ops: Interpolator[A])
+case class SurfacePointProperty[@specialized(Double, Float, Int, Boolean) A](override val triangulation: TriangleList, pointData: PointId => A)(implicit val ops: Interpolator[A])
```

Comment:
@sschoenborn  If we make it public, shouldn't we use a more descriptive name? It is not really clear (at least to me) that Ops refers to the interpolator.